.github/workflows/test-released-osx-pkg.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
           echo "version=${VERSION}" >> $GITHUB_OUTPUT
       - name: Download Package
         run: |
-          curl -SL -O https://releases.mondoo.com/mondoo/${{ steps.version.outputs.version }}/mondoo_${{ steps.version.outputs.version }}_darwin_universal.pkg
+          curl -SL -O https://releases.mondoo.com/mondoo/latest/mondoo_${{ steps.version.outputs.version }}_darwin_universal.pkg
```

Comment:
if we use latest, this will fail if the targeted version is not the right one as it will likely 404 here

Reply:
This doesn't work due to the way latest maps... this breaks testing and will require a change to the way index building works.  Please revert it.